test_scripts/Defects/7_1/1585_AlertManeuver_UNSUPPORTED_RESOURCE_response_success.lua

```diff
@@ -0,0 +1,87 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1585
+---------------------------------------------------------------------------------------------------
+-- Description: HMI responds with UNSUPPORTED_RESOURCE to Speak component of AlertManeuver
+--
+-- Preconditions:
+-- 1) Clean environment
+-- 2) SDL, HMI, Mobile session started
+-- 3) Registered app
+-- 4) Activated app
+--
+-- Steps: 
+-- 1) Send AlertManeuver mobile RPC from app with parametr imageType = "STATIC", HMI responds with 
+--    UNSUPPORTED_RESOURCE to Speak request
+--
+-- Expected:
+-- 1) App receives AlertManeuver response, with WARNINGS result code and success=true
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local params = {
+  ttsChunks = {
+    {
+      type = "TEXT",
+      text = "Text to Speak"
+    }
+  },
+  softButtons = {
+    {
+      systemAction = "DEFAULT_ACTION",
+      isHighlighted = true,
+      softButtonID = 5517,
+      type = "BOTH",
+      image = {
+        value = "icon.png",
+        imageType = "STATIC"
+      },
+      text = "Close"
+    },
+  }
+}
+
+--[[ Local Functions ]]
+local function sendAlertManeuver()
+  local mobileSession = common.mobile.getSession()
+  local hmi = common.hmi.getConnection()
+  local dataToHMI = {
+    appID = common.app.getHMIId()
+  }
+
+  local cid = mobileSession:SendRPC("AlertManeuver", params)
+  hmi:ExpectRequest("Navigation.AlertManeuver", dataToHMI)
+  :Do(function(_, data)
+    hmi:SendResponse(data.id, data.method, "SUCCESS", {})
```

Comment:
```suggestion
      hmi:SendResponse(data.id, data.method, "SUCCESS", {})
```
Please update the sequence in the function according to the issue